libs/checkpoint/langgraph/checkpoint/base/utils.py

```diff
@@ -0,0 +1,10 @@
+from typing import Optional
+
+from langchain_core.runnables import RunnableConfig
+
+
+def get_checkpoint_id(config: RunnableConfig) -> Optional[str]:
```

Comment:
should we export this from the file where base class is defined, I think makes more sense for something a lot of subclasses will want to use?

Reply:
sure